app/config/config.yml

```diff
@@ -20,6 +20,25 @@ parameters:
   mail_themes_dir: "%kernel.project_dir%%mail_themes_uri%"
   modules_translation_paths: [ ]
 
+  prestashop.mode_dev: !php/const _PS_MODE_DEV_
+  prestashop.employee_img_dir: !php/const _PS_EMPLOYEE_IMG_DIR_
+  prestashop.profile_img_dir: !php/const _PS_PROFILE_IMG_DIR_
+  prestashop.product_img_dir: !php/const _PS_PRODUCT_IMG_DIR_
+  prestashop.img: !php/const _PS_IMG_
```

Comment:
it seems that there is a problem with `_PS_IMG_`, see testsuite.

Reply:
The test suite does not boot because of this :(. I've removed it for now and it should be fixed in another PR